### DISCUSSION TITLE
Collumn Collapse calculation process question

### DISCUSSION THREAD

**Discussion by jacobosaa** [Anura3D/Anura3D_OpenSource#88] [Q&A]
Hi everyone! 

My name is Jacobo and I'm currently doing an investigation about the collapse behaviour of collapse made of tailing material, and everything is working fine, i'm following the instructions in the tutorial manual, but when i'm done with the pre-process stage in GiD, i get the CPS and GOM files. 
My problem is that, following the manual, the model develops itself in two stages: the initialization of stresses and the failure of the collumn. Everything is ok until i have to modify the CPS file from the first stage, because the tutorial says that the last CPS file  (CPS_003) is the one that has to be re-written, but at the final of the process, i only have the CPS_002. is this ok? should i re-do the model again until i get a CPS_003 or can i re-write this file the same way the tutorial indicates?
I hope you can understand my primitive english and that you can help me with this issue.
have a nice day! greetings from Chile!

**Comment by jacobosaa** [comment]
I just realized that i didn't change the number of calculations steps in GiD, but i would love to read some feedback from you so i can feel safer about this change.

**Reply by murphy-john** [reply]
The calculation will start from whatever the highest numbered CPS file. Assuming the conditions (Material Point positions, stresses, velocities, etc.) were correct after the first load step, you can change CPS_002 and the code will start from there. 

Also, your English is very good! ¡Es mucho mejor que mi español!

**Reply by jacobosaa** [reply]
Thanks john! your reply is very useful, I already fixed the issue and my model is running fine. Thank you very much for your answer, and don't worry, tu español es muy bueno!
